vb.go

```diff
@@ -0,0 +1,54 @@
+package sgbucket
+
+import (
+	"fmt"
+	"sync"
+)
+
+type VbucketNo uint32
+type VbucketSeq uint64
```

Comment:
I spent some time over the weekend experimenting with switching some of the accel code to use aliased types for this type of data (VbucketNo, VbucketSeq, PartitionNo in my case).  I ended up backing it out, because it ended up with a lot of type casting back to uint32/uint64every time I needed to call an existing function that the raw uint. 

I'm still a bit torn - it makes the code more self-descriptive in some areas, and I'm always having to pause to remember whether vbno is uint32 or uint16 off the top of my head - so I'm willing to give it a try here to see if it ends up useful.